packages/app/src/specs/SpecsListBanners.vue

```diff
@@ -96,7 +96,7 @@
     </Button>
   </Alert>
   <Alert
-    v-if="showProjectRequestAccess"
+    v-else-if="showProjectRequestAccess"
```

Comment:
I think many v-else-if combination is fine, but one thing I started using in my personal projects lately is `v-switch`, I wonder if it would be useful in Cypress, too: https://www.npmjs.com/package/@lmiller1990/v-switch

The source code is only about 10 lines, we could just copy-paste and register as a global component.